app/views/budgets/_budget.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="transaction_item">
+  
+<div class="subtitle_head">
+ <p>
+    <%= budget.budget.name %>
+  </p>
+  <p class="price">
+   $ <%= budget.budget.amount %>
+  </p>
+  </div>
+
+  
+  <div class="button_date">
+  <p>
+    <%= budget.budget.created_at %>
+  </p>
```

Comment:
- [Optional] I think the default date format doesn't really look interesting.  The format can be improved using [these ](https://stackoverflow.com/questions/29589484/how-to-change-created-at-format)suggestions

Reply:
Done!